### PR TITLE
SampleRateConverter: Improve testsuite documentation/comments

### PR DESCRIPTION
**Pull request by nbraud**
Apologies, I forgot to push this commit before you merged #324.

### CODE REVIEW

**Comment by nbraud** [comment]
PS: Once this is merged, would you be willing to label it `hacktoberfest-accepted`, along with #324 and cpal#483 ?

**Comment by est31** [comment]
@nbraud done, enjoy your t shirt :)

File: src/conversions/sample_rate.rs [hunk 1]
```diff
@@ -256,13 +256,15 @@ mod test {
     use cpal::SampleRate;
     use quickcheck::quickcheck;
 
-    // TODO: Remove once cpal::SampleRate implements ops::Mul
+    // Remove once cpal 0.12.2 is released and the dependency is updated
```
Comment (est31) [inline src/conversions/sample_rate.rs:259]: Still needs a TODO

File: src/conversions/sample_rate.rs [hunk 2]
```diff
@@ -308,6 +313,8 @@ mod test {
                        output)
         }
 
+        /// Checking that, after multiplying the sample rate by k, every k-th
```
Comment (est31) [inline src/conversions/sample_rate.rs:316]: please use Check here not Checking

**Review by est31 — CHANGES_REQUESTED** [review]
see inline comments

**Comment by nbraud** [comment]
Feedback addressed, thanks again  <3

**Review by est31 — APPROVED** [review]
Thanks!

**Pull request merged**